src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/csip/attendees/CSIPAttendeeMapping.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.csip.attendees
```

Comment:
The code is just repeated for each csip mapping -> attendee/plan/review/interview